buildconfig/CMake/PythonPackageTargetFunctions.cmake

```diff
@@ -93,8 +94,22 @@ CustomInstallLib = patch_setuptools_command('install_lib')
         # On UNIX systems install the workbench executable directly.
         # The Windows case is handled with a custom startup script installed in WindowsNSIS
         if ( NOT WIN32 )
-          install(PROGRAMS ${_setup_py_build_root}/install/bin/${_executable_name} DESTINATION bin)
+            install(PROGRAMS ${_setup_py_build_root}/install/bin/${pkg_name}
+                DESTINATION ${WORKBENCH_BIN_DIR}
+                RENAME launch_workbench)
         endif()
+
+    # register the "installed" components with cmake so it will carry them over
```

Comment:
I think this is the cause of the duplicate `lib` directory installation. Is this not a duplicate of line `87`?

Reply:
I think line `87` is the one we need to keep.